pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.api</groupId>
+  <artifactId>gax-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>2.7.1-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+  <name>GAX (Google Api eXtensions) for Java (Parent)</name>
+  <url>https://github.com/googleapis/gax-java</url>
+  <description>Google Api eXtensions for Java (Parent)</description>
+
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-shared-config</artifactId>
```

Comment:
Is this parent necessary? For what? The Gradle doesn't seem to use it? And we specify all the versions of the libraries in the dependency management section?

Reply:
This is to help us align better with client libraries. It has some checks and general configuration that is useful.

Reply:
Cool. Just wanted to make sure we don't diverge from the Bazel setup. Once we start doing that, it will be difficult to reconcile and maintain them later.

Reply:
This will technically change how gax artifacts get published on maven (they get parents which  they used to not have). 

Maybe it is ok, but we should be careful here. Can it cause any breakages and/or dependency conflicts for existing users of this? Also, I'm not sure if changing a parent can be considered a breaking change for maven atrifact or not.

Reply:
I don't think it's a breaking change. Besides, almost everything else in the Java SDK has it as the parent.